test/css/component_specific_selectors_test.rb

```diff
@@ -34,6 +34,21 @@ class ComponentSpecificSelectorsTest < Minitest::Test
     Primer::Alpha::SegmentedControl => [
       ".Button-withTooltip"
     ],
+    Primer::Beta::BorderBox => [
+      ".Box-btn-octicon",
+      ".Box--spacious .Box-title",
+      ".Box-row--unread",
+      ".Box-row.unread",
+      ".Box-row.navigation-focus",
+      ".Box-row--focus-gray",
+      ".Box-row--focus-blue",
+      ".Box-row-link",
+      ".Box-row--drag-button",
+      ".Box--scrollable",
+      ".Box--blue",
+      ".Box--danger",
+      ".Box-header--blue"
```

Comment:
Since there are quite a few un-used `.Box` styles, an alternative would be:

1. Rename `.Box` classes to `.BorderBox`. To match the `Primer::Beta::BorderBox` name.
2. Then only keep the styles that are needed for PVC.
3. On dotcom we can keep importing the `.Box` styles from PCSS until all use cases are replaced either by `BorderBox` or the upcoming `DataTable`.

I might do that on Monday.

Reply:
Ok, nvm... seems that `Primer::Beta::BorderBox` is already too dependent on some of the undocumented styles of `.Box`. Like `Box-btn-octicon` [being used inside `Primer::Beta::BorderBox`](https://github.com/search?q=repo%3Agithub%2Fgithub+Primer%3A%3ABeta%3A%3ABorderBox+Box-btn-octicon&type=code). I guess we _could_ add 

```diff
+ .BorderBox,
  .Box {
    ...
  }
```

to PCSS. But that also feels backwards and keeps things hard to maintain. So I think we should do:

1. Keep this PR "as is".
2. Then at some point, go through all the ignored `.Box` classes and decide if they should become part of the `Primer::Beta::BorderBox` API. For example [`.Box--blue` is only used once on dotcom](https://github.com/search?q=repo%3Agithub%2Fgithub+Box--blue&type=code). So we probably just want to use a blue border for that case and remove the `.Box--blue` class in PVC.